tests/runtime/conftest.py

```diff
@@ -251,6 +251,23 @@ def sample_task_model_id(good_model_path) -> str:
     model_manager.unload_model(model_id)
 
 
+@pytest.fixture
+def file_task_model_id(box_model_path) -> str:
+    """Loaded model ID using model manager load model implementation"""
+    model_id = random_test_id()
+    model_manager = ModelManager.get_instance()
+    # model load test already tests with archive - just using a model path here
+    model_manager.load_model(
+        model_id,
+        local_model_path=box_model_path,
+        model_type=Fixtures.get_good_model_type(),  # eventually we'd like to be determining the type from the model itself...
+    )
+    yield model_id
+
+    # teardown
+    model_manager.unload_model(model_id)
```

Comment:
Worth having a central function in `conftest` that does all this ? I see a lot of repeated code. Can be tackled in a subsequent PR!

Reply:
I think this is in `runtime/conftest` because it's using the runtime's `ModelManager`, right?

Reply:
Ah I meant a function within `runtime/conftest` itself to avoid repeated code, we've used the same logic to create all `*model_id`